README.md

```diff
@@ -106,7 +106,37 @@ client.addHeader(string,string);
 ---
 
 ## Development
-* Open solution in visual studio 2022, it should build fine
+
+### Quick Start
+The project now uses a unified multi-targeting approach. Simply:
+
+```bash
+# Restore dependencies
+dotnet restore
+
+# Build all target frameworks
+dotnet build --configuration Release
+
+# Create NuGet package
+dotnet pack --configuration Release
+```
+
+### Target Frameworks Supported
+- .NET Framework 4.0, 4.5, 4.7, 4.8
```

Comment:
- remove .net 4.0 and 4.5 
- add notes for 4.0 and 4.5 that people can compile the code and get the DLL